VK_App/VK_App/Controller/VKWebViewController.swift

```diff
@@ -0,0 +1,97 @@
+// VKWebViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+import WebKit
+
+/// Экран начальной страницы приложения
+final class VKWebViewController: UIViewController {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let storyboardName = "Main"
+        static let loginVCIdentifier = "loginVC"
+        static let tokenKey = "access_token"
+        static let userIdKey = "user_id"
+    }
+
+    // MARK: - Private IBOutlet
+
+    @IBOutlet private var webView: WKWebView!
+
+    // MARK: - Life Cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupWebView()
+    }
+
+    // MARK: - Private Method
+
+    private func setupWebView() {
+        webView.navigationDelegate = self
+
+        var urlComponents = URLComponents()
+        urlComponents.scheme = "https"
```

Comment:
```suggestion
        urlComponents.scheme = Constants.https
```
и все другие строки тоже должны быть в константах